DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # df[df['experiment'].str.contains("025deg_jra55v13") &amp; (df.ncfiles &gt; 500)]
</pre><p>remove commented line if you're not using it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Indeed! :)